File: Python/1 - Iniciante/1036.py
# coding: UTF - 8
# Funcional no URI

import math


numeros = input().split(" ")
A = float(numeros[0])
B = float(numeros[1])
C = float(numeros[2])

D = (B ** 2) - 4 * A * C

if D < 0 or A == 0:
    print("Impossivel calcular")
else:
    R1 = (-B + math.sqrt(D)) / (2 * A)
    R2 = (-B - math.sqrt(D)) / (2 * A)
    print(f"R1 = {R1:.5f}")
    print(f"R2 = {R2:.5f}")

File: go/1036.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var A float64
	var B float64
	var C float64

	fmt.Scan(&A)
	fmt.Scan(&B)
	fmt.Scan(&C)

	D := (B * B) - 4*A*C

	if D < 0 || A == 0 {
		fmt.Println("Impossivel calcular")
	} else {
		R1 := (-B + math.Sqrt(D)) / (2 * A)
		R2 := (-B - math.Sqrt(D)) / (2 * A)
		fmt.Printf("R1 = %.5f\n", R1)
		fmt.Printf("R2 = %.5f\n", R2)
	}
}
